src/GitUtils.ps1

```diff
@@ -65,15 +65,20 @@ function Get-GitBranch($gitDir = $(Get-GitDirectory), [Diagnostics.Stopwatch]$sw
     Invoke-Utf8ConsoleCommand {
         dbg 'Finding branch' $sw
         $r = ''; $b = ''; $c = ''
+        dbg 'Running git rev-parse' $sw;
+        $GitOutput = $(git rev-parse --symbolic-full-name --abbrev-ref --is-inside-git-dir --is-bare-repository HEAD 2>$null)
+        $IsInsideGitDir = 'true' -eq $GitOutput[0]
+        $IsBareRepository = 'true' -eq $GitOutput[1]
+        $GitRefName = $GitOutput[2]
         if (Test-Path $gitDir\rebase-merge\interactive) {
             dbg 'Found rebase-merge\interactive' $sw
             $r = '|REBASE-i'
-            $b = "$(Get-Content $gitDir\rebase-merge\head-name)"
+            $b = "$(Get-Content $gitDir\rebase-merge\head-name)" -replace 'refs/heads/', ''
```

Comment:
I didn't notice this before, but it's more correct to only replace that as a prefix: `^refs/heads/`.